packages/hydrogen-react/src/CartProvider.tsx

```diff
@@ -128,8 +129,21 @@ export function CartProvider({
   data: cart,
   cartFragment = defaultCartFragment,
   customerAccessToken,
-  countryCode = 'US',
+  countryCode,
```

Comment:
The main thing is we don't default a countryCode in the prop, instead use the shopify provider hook to get the default country code.